### DISCUSSION TITLE
WeVidi video download support

### DISCUSSION THREAD

**Discussion by TismoDito** [machineonamission/mediaforge#155] [Ideas]
I don't know if MediaForge supports this yet, but this still seems like a good feature idea for February.
I would like WeVidi video download support implemented into this great Discord bot of yours as said in the title.
If you don't know what WeVidi is, it's basically a YouTube alternative that's like Vidlii but with actually good moderation as stated in the example video URL.
Example of a WeVidi video URL: https://wevidi.net/watch/UOxjNMyp70w

**Comment by machineonamission** [comment]
The MediaForge video downloader uses [yt-dlp](https://github.com/yt-dlp/yt-dlp). Raise an issue on their repo and when it gets added to yt-dlp MediaForge will get it as well

**Reply by TismoDito** [reply]
ok
